### ISSUE TITLE
Incorrect linked github issue in 2.1.12 release notes

### ISSUE THREAD

**Issue by hostep** [magento/devdocs#1730]
Feedback on page: [/guides/v2.1/release-notes/ReleaseNotes2.1.12CE.html](http://devdocs.magento.com/guides/v2.1/release-notes/ReleaseNotes2.1.12CE.html)

Hi!

Just noticed an incorrect link over here:
```
Video and thumbnails on the Configurable Product page now work as expected. Previously, video did not play properly, and thumbnail images were not oriented correctly. GitHub-12238 Fix submitted by Chumack Roman in pull request 12558.
```

Should point to issue '12268' instead of '12238'

**UPDATE**: looking at the code in 2.1.12, this issue hasn't been fixed yet, I can't see the changes from PR 12558 in the code of 2.1.12, so maybe best to remove that part from the release notes again?

Thanks!

**Comment by jfrontain** [comment]
@hostep! Thank you for being such an excellent reviewer! I will remove this information asap. Thanks again. Will let you know when I repost the new version.

**Comment by hostep** [comment]
Thanks! :)

Also: I don't quite understand the notices about the copyright changes, do they have something to do with the code? Because I can't find really find copyright changes in the code between 2.1.11-2.1.12 or 2.2.2-2.2.3 ...

**Comment by jfrontain** [comment]
I've just merged the revised 2.1.12 release notes; it typically takes anywhere between 30 - 40 minutes for merged changes to appear on our public site. I removed the community fix you mentioned, and we've adjusted our internal JIRA issue -- thanks for catching that. 

I'm also preparing another update for the 2.0.x and 2.1.x notes to clarify how we handled the copyright update. For 2.2.3, I believe we changed one file only to handle this update. I'll push revision for all these notes to clarify this issue as soon as possible.

Thanks again, @hostep -- your comments are always valuable.
